PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -338,7 +333,25 @@ def miniAOD_customizeCommon(process):
     for idmod in photon_ids:
         setupAllVIDIdsInModule(process,idmod,setupVIDPhotonSelection,None,False,task)
 
-    #-- Adding boosted taus
+    process.patPhotons.photonIDSources=cms.PSet()
+    for egid in process.egmPhotonIDs.physicsObjectIDs:
+        setattr(process.patPhotons.photonIDSources,egid.idDefinition.idName.value(),cms.InputTag('egmPhotonIDs:'+egid.idDefinition.idName.value()))
+
+    #add the cut base IDs bitmaps of which cuts passed
+    from RecoEgamma.EgammaTools.egammaObjectModifications_tools import makeVIDBitsModifier
+    egamma_modifications.append(makeVIDBitsModifier(process,"egmGsfElectronIDs","egmPhotonIDs"))
+    
+    #e/gamma scale & smearing
+    #we will run in value map producting mode keyed to the orginal collection
+    #only enabled for 94X right now
+    if run2_miniAOD_94XFall17.isChosen():
```

Comment:
isChosen  is deprecated and should not be used.

Also, I expect that all modifications made for 94XFall17 should apply also for 80XLegacy